### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by tiserett**
- [DEMO LINK](https://tiserett.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://tiserett.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,6 +11,40 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+        class="form__field"
```
Comment (AndriiRut) [inline src/index.html:15]:
```suggestion
      class="form__field"
```
To much spaces:
image

**Review by AndriiRut — CHANGES_REQUESTED** [review]
Good job!

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-weight: 900;
```
Comment (AndriiRut) [inline src/style.css:48]: If you use such `font-weight` first of all you should import it.

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  font-size: 14px;
+  line-height: 19;
+}
+
+.search__icon {
+  position: absolute;
+  margin: 25px auto 26px 26px;
+}
+
+.search__icon_small {
```
Comment (AndriiRut) [inline src/style.css:67]:
```suggestion
.search__icon-small {
```

**Review by AndriiRut — CHANGES_REQUESTED** [review]
Good job!

**Comment by tiserett** [comment]
I changed code style and classes naming, also deleted some irrelevant code.

**Review by AndriiRut — APPROVED** [review]

**Pull request closed without merging**